pages/common/copr.md

```diff
@@ -0,0 +1,20 @@
+# copr
+
+> Interface with Fedora-Projects copr instance for building RPMs and publishing them.
+> More information: <https://developer.fedoraproject.org/deployment/copr/copr-cli.html>.
```

Comment:
```suggestion
> More information: <https://manned.org/copr>.
```
The fedoraproject link barely gives out any information on how to use it. Manned would be preferable.